### PR TITLE
Blood Drunk Miner now uses a PKA

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
## About The Pull Request

Replaces the BDM normal KA with a PKA and damage accordingly  

## Why It's Good For The Game
Makes the BDM harder to fight as well as more rewording other then a rather lame large knife.

## Changelog
:cl:
balance: BDM now uses a PKA rather then a normal KA
/:cl:

### CODE REVIEW

**Comment by Trilbyspaceclone** [comment]
I hate how if you misstakenly hit ENTER rather then shift it posts/makes the PR, but dosnt for commits...

**Comment by Arturlang** [comment]
Now its going to be fucking horrifying, nice. a really steep upgrade but its needed a buff for a good time now, plus the old pka drop was largely unused.

File: code/modules/mob/living/simple_animal/hostile/megafauna/blood_drunk_miner.dm [hunk 1]
```diff
@@ -73,7 +73,7 @@ Difficulty: Medium
 	target.stun_absorption -= "miner"
 
 /obj/item/projectile/kinetic/miner
-	damage = 20
+	damage = 50
```
Comment (silicons) [inline code/modules/mob/living/simple_animal/hostile/megafauna/blood_drunk_miner.dm]: this is a bit high

Comment (Trilbyspaceclone) [reply code/modules/mob/living/simple_animal/hostile/megafauna/blood_drunk_miner.dm]: ill lower it to 40 then

**Comment by necromanceranne** [comment]
He does still suffer from some really shitty AI that can be exploited. Nothing that can reasonably be done about that without a big overhaul. I don't think this changes the best method of fighting him, which is getting into his face and face tanking him, but it might let him snipe you if you dip a bit low so it's much more risky.

**Review by Ghommie — APPROVED** [review]

**Pull request merged**